docs/tutorial/part-two/index.md

```diff
@@ -197,7 +197,7 @@ Let's make a quick improvement. Many sites have a single column of text centered
 in the middle of the page. To create this, add the following styles to the
 `<div>` in `src/pages/index.js`.
 
-```jsx{4}
+```jsx{4,23}
```

Comment:
I'm only seeing one line of addition to the code (line 4). What other addition did you see?

Also, what would be the purpose of highlighting the closing </div>? (which seems like it's on line 25, although I could be wrong since I'm not a developer!)